internal/pkg/gitdirty/gitdirty.go

```diff
@@ -188,65 +212,78 @@ func remoteConvertHTTPStoSSH(httpsRemote string) (string, error) {
-	return strings.TrimRight(remoteUrl, ".git")
+	// Trim the git@ bc you can still have a remote url of type ssh w/o the git@
+	trimmedRemoteUrl := strings.TrimLeft(remoteUrl, "git@")
+	return strings.TrimRight(trimmedRemoteUrl, ".git")
 }
 
 // getRemoteName queries the repo at GitDirty.path for all remotes, and then
 // searches for the remote that matches the provided url, returning an error if
 // no remote url is found.
 // It will also attempt to match against different protocols - if an https protocol is
 // specified, if it can't find an exact match, it will look for an ssh-style match (and vice-versa)
-func getRemoteName(log hclog.Logger, repoPath string, remoteUrl string) (name string, err error) {
+func getRemoteName(log hclog.Logger, repoPath string, wpRemoteUrl string) (name string, err error) {
 	repo, err := git.PlainOpenWithOptions(repoPath, &git.PlainOpenOptions{
 		DetectDotGit: true,
 	})
 	if err != nil {
 		return "", errors.Wrapf(err, "failed to open git repo at path %q", repoPath)
 	}
 
-	remotes, err := repo.Remotes()
+	localRepoRemotes, err := repo.Remotes()
 	if err != nil {
-		return "", errors.Wrap(err, "failed to list remotes")
+		return "", errors.Wrap(err, "failed to list localRepoRemotes")
 	}
 
-	if len(remotes) == 0 {
-		return "", fmt.Errorf("no remotes found for repo at path %q", repoPath)
+	if len(localRepoRemotes) == 0 {
+		return "", fmt.Errorf("no localRepoRemotes found for repo at path %q", repoPath)
 	}
 
 	var exactMatchRemoteName string
-	for _, remote := range remotes {
-		remoteConfig := remote.Config()
-		if remoteConfig == nil {
+	for _, localRepoRemote := range localRepoRemotes {
+		localRepoRemoteConfig := localRepoRemote.Config()
+		if localRepoRemoteConfig == nil {
 			continue
 		}
-		if len(remoteConfig.Fetch) == 0 {
+		if len(localRepoRemoteConfig.Fetch) == 0 {
 			// Must be able to fetch from the remote. This could happen if a remote is set up as a push mirror.
 			continue
 		}
-		for _, thisRemoteUrl := range remoteConfig.URLs {
-			if normalizeRemote(thisRemoteUrl) == normalizeRemote(remoteUrl) {
+		for _, localRemoteUrl := range localRepoRemoteConfig.URLs {
+			if normalizeRemote(localRemoteUrl) == normalizeRemote(wpRemoteUrl) {
 				if exactMatchRemoteName != "" {
-					// NOTE(izaak): I can't think of a dev setup where you'd get multiple remotes with the same url.
+					// NOTE(izaak): I can't think of a dev setup where you'd get multiple localRepoRemotes with the same url.
 					// If it does though, I think it's likely that any remote will work for us for diffing purposes,
 					// wo we'll warn and continue.
-					log.Warn("Found multiple remotes with the target url. Will choose remote-1.", "url", thisRemoteUrl, "remote-1", exactMatchRemoteName, "remote-2", remoteConfig.Name)
+					log.Warn("Found multiple localRepoRemotes with the target url. Will choose remote-1.", "url", localRemoteUrl, "remote-1", exactMatchRemoteName, "remote-2", localRepoRemoteConfig.Name)
```

Comment:
Another example of where "remotes" makes more sense as an error message rather than "localRepoRemotes"